code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -359,15 +358,17 @@
 	for(var/obj/item/organ/O in contents)
 		O.Insert(C)
 
+	for(var/i in wounds)
+		var/datum/wound/W = i
+		W.apply_wound(src, TRUE)
```

Comment:
I'm a bit confused by this logic.
This copies the limb's list of wounds first.
Then it goes to every wound and tells them to re-apply themselves to the limb.
Aren't they caught here and delete themselves, in `apply_wound()`?
```cpp
	for(var/i in L.wounds)
		var/datum/wound/preexisting_wound = i
		if((preexisting_wound.type == type) && (preexisting_wound != old_wound))
			qdel(src)
			return
```